src/app/app.component.html

```diff
@@ -1 +1,142 @@
+                        </ul>
+                    </div>
+                </div>
+            </div>
+            <div class="col-12 col-md-12 col-lg-4">
+                <div class="row g-4 pb-4">
+                    <div class="col-12">
+                        <h6 class="mb-4">Stores</h6>
+                        <ul class="nav flex-column">
+                            <li *ngFor="let store of storeList$ | async" class="nav-item"><a [routerLink]="'stores/'+store.id"
+                                    class="nav-link">{{store.name}}</a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+            <div class="col-12 col-md-12 col-lg-4">
+                <div class="row g-4">
+                    <div class="col-12">
+                        <h6 class="mb-4">Get to know us</h6>
+                        <ul class="nav flex-column">
+                            <li *ngFor="let item of getToKnowUsOptionList$ | async" class="nav-item">{{item}}</li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="border-top py-4">
+            <div class="row align-items-center">
+                <div class="col-lg-5 text-lg-start text-center mb-2 mb-lg-0">
+                    <ul class="list-inline mb-0">
+                        <li class="list-inline-item text-dark">Payment Partners</li>
+                        <li class="list-inline-item"> <a href="#!"><img src="../assets/images/payment/amazonpay.svg" alt=""></a> </li>
+                        <li class="list-inline-item"> <a href="#!"><img src="../assets/images/payment/american-express.svg" alt=""></a>
+                        </li>
+                        <li class="list-inline-item"> <a href="#!"><img src="../assets/images/payment/mastercard.svg" alt=""></a> </li>
+                        <li class="list-inline-item"> <a href="#!"><img src="../assets/images/payment/paypal.svg" alt=""></a> </li>
+                        <li class="list-inline-item"> <a href="#!"><img src="../assets/images/payment/visa.svg" alt=""></a> </li>
+                    </ul>
+                </div>
+                <div class="col-lg-7 mt-4 mt-md-0">
+                    <ul class="list-inline mb-0 text-lg-end text-center">
+                        <li class="list-inline-item mb-2 mb-md-0 text-dark">Get deliveries with FreshCart</li>
+                        <li class="list-inline-item ms-4"> <a href="#!"> <img src="../assets/images/appbutton/appstore-btn.svg" alt=""
+                                    style="width: 140px;"></a> </li>
+                        <li class="list-inline-item"> <a href="#!"> <img src="../assets/images/appbutton/googleplay-btn.svg" alt=""
+                                    style="width: 140px;"></a> </li>
```

Comment:
no inline styles, use existing bootstrap utility classes

Reply:
I will add those svg graphics later with proper classes because they are not displaying in the right way and are not crucial at this point.

For now they are just deleted.